fog/sample-paykit/src/client.rs

```diff
@@ -183,17 +198,22 @@ impl Client {
     /// transaction.
     pub fn send_transaction(&mut self, transaction: &Tx) -> Result<u64> {
         let start_time = std::time::SystemTime::now();
-        let block_count = self.consensus_service_conn.propose_tx(transaction)?;
+        let fee_map = self.get_latest_fee_map()?;
+        let tx_ok_data = self.consensus_service_conn.propose_tx_v2(transaction, &fee_map).map_err(|err| {
+            // Clear fee map in case this was the problem
+            self.fee_map = None;
```

Comment:
Note: in a real client, to defend against the attack, what you should actually do is check the fee map with multiple consensus nodes. if they don't agree then something bad is happening to the network, either maliciously or not. nodes that dont agree about the fee map necessarily fail to consense to one another.

Reply:
I guess I should add a code comment like this